### DISCUSSION TITLE
Chapters as Headings

### DISCUSSION THREAD

**Discussion by erankatz2020** [hadynz/obsidian-kindle-plugin#233] [Ideas]
It would be great if we could have automatic chapter headings in the highlights note. Maybe there's a way to manually highlight the chapter title in Kindle and add something to its not (such as .h2. or something) and have the plugin recognize that and convert the highlighted text into a H2 heading. 

Is that possible?

**Comment by youknowmeasnewt** [comment]
If this plugin controls highlight colors, it could make it possible.
